flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/batch/sql/MultipleInputCreationTest.scala

```diff
@@ -355,5 +355,5 @@ object MultipleInputCreationTest {
 
   @Parameters(name = "shuffleMode: {0}")
   def parameters: Array[BatchShuffleMode] =
-    Array(BatchShuffleMode.ALL_EXCHANGES_BLOCKING, BatchShuffleMode.ALL_EXCHANGES_PIPELINED)
```

Comment:
I do think we should remove BatchShuffleMode.ALL_EXCHANGES_PIPELINED,  We can defined the parameter as 
```
  def parameters(): util.Collection[Array[java.lang.Object]] = {
    Seq[Array[AnyRef]](
      Array(BatchShuffleMode.ALL_EXCHANGES_BLOCKING, adaptive_scheduler_enabled),
      Array(BatchShuffleMode.ALL_EXCHANGES_BLOCKING, adaptive_scheduler_disabled),
      Array(BatchShuffleMode.ALL_EXCHANGES_PIPELINED, adaptive_scheduler_disabled),
    )
}
```

Reply:
Fixed it in the proposed way.